docker/docker_client.go

```diff
@@ -533,7 +548,7 @@ func (c *dockerClient) detectProperties(ctx context.Context) error {
 			return true
 		}
 		isV1 := pingV1("https")
-		if !isV1 && c.sys != nil && c.sys.DockerInsecureSkipTLSVerify {
+		if !isV1 && c.sys != nil && c.sys.DockerInsecureSkipTLSVerify == types.OptionalBoolTrue {
```

Comment:
The two uses of `DockerInsecureSkipTLSVerify` should use `sysregistriesv2` as well (presumably looking up the config only once).

Reply:
(only once == when creating the `dockerClient`, not once in `newDockerClient…` and once in `detectProperties`.)

Reply:
A shame I didn't see this. Fixed now.

Reply:
>  Fixed now.

(I still see the same version of the code.)